### PR TITLE
GHES v1 :) 

### PR DESCRIPTION
**Pull request by NickLiffen**

### CODE REVIEW

File: src/utils/worker.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const worker = async (): Promise<unknown> => {
     const { repo, enableDependabot, enableSecretScanning, createIssue } =
       repos[index];
     await enableGHAS(repo, client);
-    if (enableDependabot) {
+    if (enableDependabot || process.env.GHES == "true") {
```
Comment (tonycch) [inline src/utils/worker.ts:26]: So we always enable Dependabot if we are working on GHES? Don't we want to skip it as the API is not supported on server?

Comment (NickLiffen) [reply src/utils/worker.ts:26]: ah this should be `process.env.GHES != "true"` 🤦

**Review by tonycch — APPROVED** [review]
Great work! Please 🚢  it

**Comment by NickLiffen** [comment]
closes #24

**Pull request merged**